src/pages/Login/style.css

```
@@ -0,0 +1,66 @@
+html {
+  font-size: 18px;
+}
+
+.center {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+.logo {
+  width: 8.4rem;
+}
+
+.login-box {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin-top: 1.8rem;
+  padding: 1.6rem 2.8rem;
+  border: 2px solid #BDBDBD;
+  color: rgba(0, 0, 0, .75);
+}
+
+  .login-input-container {
+    width: 36rem;
+    height: 7.1rem;
+  }
+
+    .login-input-row {
+      width: auto;
+      margin-bottom: .5rem;
+    }
+
+      .login-input-label {
+        display: inline-block;
+        width: 5.4rem;
+        font-size: 1rem;
+      }
+
+      .login-input-input {
+        width: 28rem;
+        padding: 1rem;
+        background-color: rgb(0, 0, 0, .05);
+        border: 1px solid #BDBDBD;
+        font-size: 1rem;
+      }
```

실행 환경에 따라 box-sizing 기준이 달라져서 input 태그의 크기가 달라집니다.
box-sizing을 명시해야 합니다.

```suggestion
      .login-input-input {
        width: 30rem;
        padding: 1rem;
        box-sizing: border-box;
        background-color: rgb(0, 0, 0, .05);
        border: 1px solid #BDBDBD;
        font-size: 1rem;
      }
```